pkgs/development/python-modules/prometheus-pandas/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, pkgs
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
+    version = "0.3.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-zysdlYPjvhextAfW4vvGN6mSPVUbWLV+AvQ1hGWCAbw=";
+    };
+
+    propagatedBuildInputs = [
+      ipython
+      matplotlib
+      numpy
+      pandas
+      requests
+    ];
+    # Tests never work. :(
```

Comment:
Maybe this comment could be a bit more descriptive, saying roughly *why* it does not work? Network based tests? Distributed computing tests? Or are they just broken?